### ISSUE TITLE
TLS/SSL support

### ISSUE THREAD

**Issue by baijerry** [sandro-k/MQTTLensChromeApp#8]
Hi, are there plans to impliment tls/ssl with this app?

**Comment by sandro-k** [comment]
Yes there is, currently I am working on a update/port to Polymer 1.0 https://github.com/sandro-k/MQTTLensChromeApp/tree/polymer_v1 and then would switch to https://github.com/mqttjs/mqtt-elements, wich already has support for TLS/SSL

**Comment by stoicskyline** [comment]
@sandro-k were you able to make any progress on TLS/SSL support?

**Comment by sandro-k** [comment]
The short answer is no.. As of https://blog.chromium.org/2016/08/from-chrome-apps-to-web.html Chrome Apps are deprecated and will not work on Mac, Windows or Linux after the second half of 2017. That is why I need to port MQTTLens to Atom or something else that runs HTML and JavaScript as a desktop application. When switching to Atom it should be possible to just use MQTT.js without the need to bundle it via Browserify and use the native SSL/TLS support of the library.
